images/bridge2/assistant/assistant.go

```diff
@@ -0,0 +1,144 @@
+package assistant
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/ajbouh/substrate/images/bridge2/tracks"
+	"github.com/ajbouh/substrate/images/bridge2/transcribe"
+)
+
+var recordAssistant = tracks.EventRecorder[*AssistantEvent]("assistant")
+
+type AssistantEvent struct {
+	SourceEvent tracks.ID
+	Name        string
+	Prompt      string
+	Response    string
+	Error       string
+}
+
+type Agent struct {
+	Assistants map[string]string
```

Comment:
what is this a map of

Reply:
It's currently a map from name to endpoint.
Though based on the other comment about the system prompts, I think I'll restructure this a bit to make separate "assistant" structs with the relevant configurations for endpoint, prompt, etc.

Reply:
got it, sounds good